migrate/v0_15/migrate.md

```diff
@@ -0,0 +1,91 @@
+# kava-8 Upgrade Instructions
+
+## Software Version and Key Dates
+
+- We will be upgrading from chain-id "kava-7" to chain-id "kava-8".
+- The version of Kava for kava-8 is [TODO: version]
+- Kava-7 validators should prepare to shutdown their nodes August 30th, 2021 at 13:00 UTC by setting `--halt-time` to `1630328400`
+- kava-7 genesis time is set to August 30th, 2021 at 15:00 UTC
+- The version of cosmos-sdk for kava-8 is v0.39.2
```

Comment:
v0.39.3

Reply:
so it is a different version than kava-7?